images/linux/ubuntu1604.json

```diff
@@ -129,6 +129,15 @@
             ],
             "execute_command": "sudo sh -c '{{ .Vars }} {{ .Path }}'"
         },
+        {
+            "type": "shell",
+            "inline": [
```

Comment:
Can we move this to a script that's shared for both images? I would also include the inline calls above to set ImageVersion and ImageOS. I rather we minimize the logic directly in the json file since it's harder to test locally.

Reply:
Sure thing! I'll make the changes.

Reply:
Changes have been made